### PR TITLE
[Issue 28] - Docker images for frontend

### LINKED ISSUE #28: update UI configuration for running locally
Update the front end UI to allow running UI locally.

### PR DESCRIPTION
**Pull request by natesulat**
* `Dockerfile` added for frontend UI package and the fingerprint scanner simulator
* Created `docker-compose` configs for the `ssirius` and `fingerprint-simulator` services
* Created `frontend` profile in the `docker-compose` config
* Added Playwright as a devDependency
* Added `@kiva/fingerprint-api-simulator` as a devDependency
* Updated `README` with instructions on how to build the frontend

### CODE REVIEW

**Review by matt-raffel-kiva — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -14,18 +14,21 @@
     "test": "node -r dotenv/config node_modules/.bin/jest --runInBand",
     "test:concurrency": "node -r dotenv/config node_modules/.bin/jest --testRegex concurrency",
     "test:identity": "node -r dotenv/config node_modules/.bin/jest --testRegex identity",
-    "test:remote": "node -r dotenv/config node_modules/.bin/jest --runInBand --testRegex remote"
+    "test:remote": "node -r dotenv/config node_modules/.bin/jest --runInBand --testRegex remote",
+    "fpapi": "swrl"
```
Comment (ghost) [inline package.json]: What does this command do? If possible, it would be nice to use full words for this.

**Review by ghost — COMMENTED** [review]
I guess I'm a little confused as to why the Dockerfiles are in this repo? The pattern we typically follow is to build the docker image in the original repo using the Dockerfile in that repo, push it to dockerhub, and then pull down that image here. It makes the tests run faster and keeps that original repo as the source of truth for facts about that repo, maintaining proper separation of concerns.

If there's a good reason that they don't belong in their original repo and do belong here, I'm not necessarily opposed. I just don't understand why the patterns is being broken.

**Comment by matt-raffel-kiva** [comment]
> I guess I'm a little confused as to why the Dockerfiles are in this repo?

Part of the reason is we do not have a clear place for stand alone installer changes.  See the "standalone-installer" branch as another example.

Its been the backend design that docker images reside in the repo they use.  This has not applied to front end design-- the front end systems doesn't use docker.   The docker images are for the stand alone install work.

Lets have a meeting about this rather than continued conversations here so that we get it resolved.

**Comment by ghost** [comment]
> Lets have a meeting about this rather than continued conversations here so that we get it resolved

Sounds good.

File: package.json (hunk 1 above)
Comment (natesulat) [reply package.json]: It runs the fingerprint api simulator. Happy to change it to:

```
"run:fingerprint-api-simulator": "swrl"
```

**Comment by natesulat** [comment]
@jeffk-kiva There's a precedent for this in the `services/indy-pool` directory, so I based it off that.

I think we can consider this an alpha stage - ultimately, publishing the images to Docker Hub will be good, if for no other reason than because it takes a long time for the frontend to install, so if we could have a pre-built image for that I'd feel better.

But publishing an image that doesn't work to Docker Hub is not what I had in mind. This PR is meant to prove that it can be done before we go that route.

File: package.json (hunk 1 above)
Comment (ghost) [reply package.json]: Aha yeah, I see why you shortened it. Maybe just `"fingerprintApi": "swrl"` so the full command reads `npm run fingerprintApi`

Comment (natesulat) [reply package.json]: Deal
RocketSnail

**Comment by ghost** [comment]
@natesulat that makes sense to me. Just make sure there's a ticket for the follow-up work to deploy to dockerhub and remove it from here and I'm cool with it. FWIW we need to do the same thing with indy-pool, it's just something I haven't gotten around to.

**Review by ghost — APPROVED** [review]

**Comment by natesulat** [comment]
Done:
https://github.com/kiva/protocol-integration-tests/issues/35
https://github.com/kiva/protocol-integration-tests/issues/36

**Pull request merged**